source-browser.html

```diff
@@ -181,7 +209,46 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           .focus();
       }
 
+      function createNavbar(topic) {
+        $('.li').append(
+          $('<a>', {
+            id: `li${topic}`,
+            href: `#${topic}`
+          }).text(topic)
+        );
+      }
+
+      let list = [];
```

Comment:
As I mentioned earlier, variable names that are just the type are undesirable in most languages unless it's a utility function with no context.